substrate/bin/minimal/node/Cargo.toml

```diff
@@ -0,0 +1,56 @@
+[package]
+name = "minimal-node"
+version = "4.0.0-dev"
+description = "A fresh FRAME-based Substrate node, ready for hacking."
```

Comment:
It this supposed to be used by anyone?  
Otherwise we could think about moving it into the docs folder, or an example folder.

Reply:
I want to use all templates under one roof, probably `developer_hub`. For now let's leave as-is? it will be addressed in https://github.com/paritytech/polkadot-sdk/compare/kiz-frame-api...kiz-umbrella-crates?expand=1